components/edit/scriptNav.tsx

```diff
@@ -74,8 +81,10 @@ const ScriptNav: React.FC<ScriptNavProps> = ({ collapsed, setCollapsed }) => {
             stringify(script).then((gptscript) => {
               navigator.clipboard.writeText(gptscript);
             });
+          } else if (key === 'new') {
+            handleNew();
           } else {
-            window.location.href = `/edit?file=${key}`;
+            window.location.href = `/edit?id=${key}`;
```

Comment:
same here

Reply:
Banged my head against this for a little bit but this doesn't reload the page so switching this would require a bit of a refactor. I'd like to get this bug fix and potentially address that after but what do you think? The issue is the pushing to this does not reload the page. There are a number of ways to fix that but I couldn't think of anything easy without calling router.push and the refreshing right after. Otherwise this doesn't actually update the state of the page.

Reply:
I pushed a partial solution to your original request along with this comment. You'll see we now use router.push() in the Create component.